src/converters/converter-manager.js

```diff
@@ -51,16 +62,40 @@ var converterManager = {
 
     list: [],
     /**
-     * Add a new attribute converter
-     * @param  {string|function|regex} alias formatter name
-     * @param  {function|object} converter    converter can either be a function, which will be called with the value, or an object with {alias: '', parse: $.noop, convert: $.noop}
-     * @param {Boolean} acceptList decides if the converter is a 'list' converter or not; list converters take in arrays as inputs, others expect single values.
+     * Add a new attribute converter to this instance of Flow.js.
+     *
+     * **Example**
+     *
+     *      Flow.dom.converters.register('max', function (value) {
```

Comment:
this example should have a "True" at the end since you're registering this to take in arrays